### ISSUE TITLE
Feature request: allow to modify file names for output

### ISSUE THREAD

**Issue by ashipunov** [trufanov-nok/scantailor#46]
From ru-board.
[My opinion: useful but not so necessary. If, for example, before Scan Tailor all files are named with prefix-001 scheme, then I see no difficulties with output names.]

**Comment by jjYBdx4IL** [comment]
I second that. the ordering of output files by name is essential for further processing. Optional prefixing of output files is essential.

**Comment by trufanov-nok** [comment]
jjYBdx4IL currently there is a workaround that should be implemented in a latest version of STU. It allows to completely overwrite output file filename: Go to Output processing stage, right click the page at page thumbnails list and choose "Overwrite output filename...". That helps to manually fix page order after insertion of a new page.

**Comment by jjYBdx4IL** [comment]
Do I need to do this for every page by hand?

**Comment by trufanov-nok** [comment]
Yes. It's not an option if you need to rename whole files in a project. But if you inserted a couple of pages that breaks your output files order it may be useful.

**Comment by jjYBdx4IL** [comment]
That's not very useful. A simple option to enable using page numbers as output file name prefix would be much more useful.
